lnode-src/duckcoin.go

```diff
@@ -227,8 +227,19 @@ func handleGetBlocks(w http.ResponseWriter, r *http.Request) {
 	io.Copy(w, f)
 	f.Close()
 }
+
+func createFloatOfBalance(bal int) float64 {
```

Comment:
A separate func is not needed, just have a global variable called “duckToQuacks” and set it to `1e8`

Reply:
This also may need us to use int64s everywhere a duckcoin amount is used.

Reply:
that'd be a good idea, I'll refactor it

Reply:
Fixed!